### PR TITLE
LPS-97526 Fix experiment creation service path

### PR DESCRIPTION
**Pull request by darquesdev**

### CODE REVIEW

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/js/reducers/segmentsExperiments.es.js [hunk 1]
```diff
@@ -29,7 +29,7 @@ export function createSegmentsExperimentsReducer(state, action) {
 function _createExperiment({name, description, segmentsExperienceId}) {
 	return new Promise(function(resolve, reject) {
 		Liferay.Service(
-			'/segments.segmentsexperiment/add-segments-experience',
+			'/segments.segmentsexperiment/add-segments-experiment',
```
Comment (darquesdev) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/js/reducers/segmentsExperiments.es.js:32]: 😰 @andresfulla

**Comment by darquesdev** [comment]
ci:test:relevant

**Comment by darquesdev** [comment]
ci:test:sf

**Comment by andresfulla** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/js/reducers/segmentsExperiments.es.js (hunk 1 above)
Comment (andresfulla) [reply modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/js/reducers/segmentsExperiments.es.js:32]: ouch!
also, is `/segments.segmentsexperiment` fine in singular?

**Pull request closed without merging**